pkg/frontend/querymiddleware/model.proto

```diff
@@ -116,3 +116,25 @@ message Hints {
 message QueryStatistics {
   uint64 EstimatedSeriesCount = 1;
 }
+
+// CachedHTTPResponse holds a generic HTTP response in the query results cache.
+message CachedHTTPResponse  {
+  // CacheKey contains the non-hashed cache key, used to guarantee there haven't
+  // been any hashed cache key collision.
+  string CacheKey = 1;
```

Comment:
nit: Fields in protobuf should be lowercase. Although some (not all) messages in this file don't do that, let's not repeat that mistake.

Applies to `CachedHTTPHeader` too.

Reply:
👍 TIL

Reply:
https://protobuf.dev/programming-guides/style/#message-field-names :)

Reply:
Aha! It's lower_snake_case for fields. Fixed in https://github.com/grafana/mimir/pull/5212/commits/6e436ed5c18499c2071c738a770f0c3f80ba043c